CMR/python/demos/collections.ipynb

```diff
@@ -20,12 +20,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
-    "sys.path.append('/Users/tacherr1/src/project/eo-metadata-tools/CMR/python')"
+    "import sys, os\n",
+    "sys.path.append(os.path.expanduser('~/src/project/eo-meta-tools-demo/CMR/python'))"
```

Comment:
use of the OS library is pretty nifty here - but why did the repo name change?

Reply:
I called the directory 'eo-meta-tools-demo' because I have two checkouts, one I do the actual development on, and another set to master for doing demos. The path is not relevant, but I could add a comment that the path is where you checked it out. Actually since this change is to load the file as a pip lib, I should remove this all together